src/common/components/cards/combined-report-result-section-title.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import * as React from 'react';
+import { OutcomeType } from 'reports/components/outcome-type';
+
+import * as styles from './combined-report-result-section-title.scss';
+
+export type CombinedReportResultSectionTitleProps = {
+    title: string;
+    badgeCount: number;
```

Comment:
Since there's no badge, could we name this something more generic like `outcomeCount`?